### PR TITLE
replace cancel actions with breadcrumbs

### PR DESCRIPTION
**Pull request by der-john**
This is part of the redesign. Breadcrumbs get introduced - in this PR, (solitary) cancel actions everywhere outside of Mercator get replaced by such 'cancel' breadcrumbs.

Besides this: breadcrumbs plus dropdown now constitute `resource-navigation` - they get renamed in the first commit. The navigation will move outside of the `resource-header` asap.

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/DebateWorkbench/DebateWorkbench.ts [hunk 1]
```diff
@@ -53,7 +53,8 @@ export var documentDetailColumnDirective = (
 
 export var documentCreateColumnDirective = (
     adhConfig : AdhConfig.IService,
-    adhTopLevelState : AdhTopLevelState.Service
+    adhTopLevelState : AdhTopLevelState.Service,
+    adhResourceUrl
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/DebateWorkbench/DebateWorkbench.ts]: This is not required, or am I missing something?

Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/DebateWorkbench/DebateWorkbench.ts]: I introduced it in the template.

Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/DebateWorkbench/DebateWorkbench.ts]: Then it is required in as a module dependency in Module.ts, but it is not necessary to inject it here.

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Image/Upload.html [hunk 2]
```diff
@@ -1,3 +1,8 @@
+<div class="resource-navigation">
+    <div class="resource-navigation-breadcrumbs">
+        <a data-ng-href="{{cancelUrl}}">{{"TR__CANCEL" | translate}}</a>
+    </div>
+</div>
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Image/Upload.html]: This directive is also used on the user profile and was not meant to be used as a page on its own (the solution for euth is a hack really).

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_resource_header.scss [hunk 3]
```diff
@@ -45,6 +45,7 @@ and a primary action (e.g. create proposal or like).
     color: $color-text-introvert;
     font-weight: bold;
     list-style: none;
+    padding: 0;
```
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_resource_header.scss]: (unrelated.)

**Comment by xi** [comment]
The cancel action uses `goToCameFrom`. This behavior should be kept.

**Comment by der-john** [comment]
this is being rebased a little.
